service.html

```diff
@@ -1,337 +1,341 @@
+                                        fill="#222222"
+                                    ></path>
+                                </svg>
                             </div>
+                        </div>
 
-                    </div>
-
-                    <!-- Copyright -->
-                    <div class="ons-grid"> 
-                        <div class="ons-grid__col ons-u-mt-s">
-                            <p class="ons-u-fs-s ons-u-mb-no">&copy; Crown copyright and database rights 2020 OS 100019153. <br> Use of address data is subject to the terms and conditions.</p>
+                        <!-- Copyright -->
+                        <div class="ons-grid">
+                            <div class="ons-grid__col ons-u-mt-s">
+                                <p class="ons-u-fs-s ons-u-mb-no">
+                                    &copy; Crown copyright and database rights 2020 OS 100019153. <br />
+                                    Use of address data is subject to the terms and conditions.
+                                </p>
+                            </div>
                         </div>
                     </div>
-
-            </div>
+                </div>
+            </footer>
         </div>
-    </footer>
-
-
-            </div>
-
 
         <script nonce="d70E02spRbfOevCK">
-            (function() {
-                var s = 'https://cdn.ons.gov.uk/sdc/design-system/44.0.2/scripts/main.js'.split(','),
-                c = document.createElement('script');
+            (function () {
+                var s = "https://cdn.ons.gov.uk/sdc/design-system/44.0.2/scripts/main.js".split(","),
+                    c = document.createElement("script");
 
-                if (!('noModule' in c)) {
+                if (!("noModule" in c)) {
                     for (var i = 0; i < s.length; i++) {
-                        s[i] = s[i].replace('.js', '.es5.js');
+                        s[i] = s[i].replace(".js", ".es5.js");
                     }
                 }
 
                 for (var i = 0; i < s.length; i++) {
-                    var e = document.createElement('script');
+                    var e = document.createElement("script");
```

Comment:
I don't know how much we are allowed to modify this code but we should change `var` to `let` to avoid duplicate declaration of `i`. Ideally, all of them should be declared with `let`. If compatibility is an issue ignore this comment.

Reply:
I would keep it as is, as it is what we do in runner and any JS used should be transpiled to es5. IE should load in `main.es5.js`